LICENCE

```diff
@@ -0,0 +1,8 @@
+Copyright © 2020, PeopleDoc Inc.
```

Comment:
@benoitbryon raised a good point. At least, let's keep @Natim's copyright since he started the project.

```suggestion
Copyright © 2012 Rémy HUBSCHER
Copyright © 2020, PeopleDoc Inc.
```

Reply:
However, please rename the file `LICENCE` → `LICENSE`. :)